packages/utils/src/lib/execute-process.ts

```diff
@@ -81,6 +81,7 @@ export type ProcessConfig = {
   command: string;
   args?: string[];
   cwd?: string;
+  shell?: boolean;
```

Comment:
I don't see this used anywhere. What is the purpose of this?

Reply:
nice catch THX!! Shouldnt there be a linting rule for this?